components/atoms/Text/TextP.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledP from './style';
+
+const TextP = ({children}) => {
```

Comment:
Can we create one common component for heading and pass p or h5 as a component in it. styling can be appended based on the passed heading type.